pkg/awsutils/awsutils.go

```diff
@@ -655,6 +665,14 @@ func (cache *EC2InstanceMetadataCache) getENIMetadata(eniMAC string) (ENIMetadat
 				log.Debugf("Found IPv6 addresses associated with interface. This is not efa-only interface")
 			}
 		}
+		if field == "network-card" {
+			networkCard, err = cache.imds.GetNetworkCard(ctx, eniMAC)
+			if err != nil {
+				awsAPIErrInc("GetNetworkCard", err)
+				log.Errorf("Network Card data not found from", networkCard)
```

Comment:
nit - format specifier missing. And we should log error, not `networkCard`

Reply:
What format specifier missing you're referring to? From existing code, it seems we always return error to upstream and handled by GetAttachedENIs() method. I think it's ok to follow existing pattern?

Reply:
log.Errorf("Network Card data not found from %v",  err)

Reply:
Something like this^

Reply:
The thing is, error is passed above to [here](https://github.com/aws/amazon-vpc-cni-k8s/blob/4cf248870605ac8eec5030f6bfd7927b773cdf05/pkg/awsutils/awsutils.go#L590) and then handled [here](
https://github.com/aws/amazon-vpc-cni-k8s/blob/4cf248870605ac8eec5030f6bfd7927b773cdf05/pkg/ipamd/ipamd.go#L1375) with logging of the error messages. We're not lostign track of the error. Is duplicating a error logging helps here? Also the above error handling is following this similar pattern.

Reply:
The only thing missing is format specifier "%v" is what I was calling out before.

Reply:
Updated in latest revision